### ISSUE TITLE
Still struggling to get this to run under Arch Linux

### ISSUE THREAD

**Issue by Toughbook-CFC2** [sultim-t/xash-rt#152]
Hi, thank you for your amazing work.
I'm very keen on playing the game with this mod. Unfortunately I can't get the game to start yet.
I created a new wine environment using winetricks, only installed vcrun2019 in there and 
because I have an AMD Vega M GH I also tried to set the corresponding environment variables:
```
env RADV_PERFTEST=rt,emulate_rt WINEPREFIX=/home/per/.local/share/wineprefixes/Test/ wine rt_bin/xash3d.exe
```
I'm sure it is obvious, but what am I doing wrong ?
Thank you for your support!

(Link to the error message: https://imgur.com/a/BkASemC)

**Comment by Nord1cWarr1or** [comment]
Try PortProton instead of Wine.
My distribution is Garuda Linux (Arch based) and I ran HL RTX without problems the first time using PortProton.

**Comment by Toughbook-CFC2** [comment]
Hey Thanks for the feedback, I tried it using PortProton and I got further then I got before. Now I constantly have the error message: 'host_initerror: Cant find memory type for given memory property flags (6)' as some others here do as well.
Could you explain what your configuration is ? Am I missing some Setting or version of Wine/Proton ?
Thank you for your help!

**Comment by Nord1cWarr1or** [comment]
> Hey Thanks for the feedback, I tried it using PortProton and I got further then I got before. Now I constantly have the error message: 'host_initerror: Cant find memory type for given memory property flags (6)' as some others here do as well. Could you explain what your configuration is ? Am I missing some Setting or version of Wine/Proton ? Thank you for your help!

Sorry, but I didn't do anything special. Just installed the HL RTX according to the instructions. And started the game with PortProton.
WINE Version (in PP) I set to "PROTON_LG_7-49".
Here is settings:
![изображение](https://user-images.githubusercontent.com/47604048/225830632-033bf4ab-786b-417e-bbd0-955d6f47be27.png)

Try to check all dependencies for PP.

**Comment by addeps3** [comment]
> > Hey Thanks for the feedback, I tried it using PortProton and I got further then I got before. Now I constantly have the error message: 'host_initerror: Cant find memory type for given memory property flags (6)' as some others here do as well. Could you explain what your configuration is ? Am I missing some Setting or version of Wine/Proton ? Thank you for your help!
> 
> Sorry, but I didn't do anything special. Just installed the HL RTX according to the instructions. And started the game with PortProton. WINE Version (in PP) I set to "PROTON_LG_7-49". Here is settings: ![изображение](https://user-images.githubusercontent.com/47604048/225830632-033bf4ab-786b-417e-bbd0-955d6f47be27.png)
> 
> Try to check all dependencies for PP.

How do you mean that you started the _game_ with PortProton? 
All I see in PortProton, regarding HL RTX, is how to start Steam, which I tried but Steam needs me to log in.

**Comment by 0x4E69676874466F78** [comment]
> Due to political times and Russia currently not being popular means Im not that excited about entering my steam credentials on a prefix coded up there.

Чел... Вальва пока не замечены среди поехавших головой. 
Если ты такой параноик тебе надо запускать всё это в песочницах, а не бояться ввести свои данные в стиме.

**Comment by addeps3** [comment]
> > Due to political times and Russia currently not being popular means Im not that excited about entering my steam credentials on a prefix coded up there.
> 
> Чел... Вальва пока не замечены среди поехавших головой. Если ты такой параноик тебе надо запускать всё это в песочницах, а не бояться ввести свои данные в стиме.

Hi. I ran your words in google transale, so I will respond to the translation, which is prolly incorrect :)

"People ... Valva has not yet been seen among those who went headlong.
If you're that paranoid, you should be running all of this in sandboxes instead of being afraid to enter your steam data."

Im not a software expert, I just hesitated when Steam asked for credentials, because Steam was running _through_ PortProton. 
Im not saying Im right, just expressing my possibly, irrational feelings about it. Thank you sharing your opinion :)

**Comment by 0x4E69676874466F78** [comment]
@addeps3 I guess I just misunderstood you. Also the translation was wrong (but it doesn't matter, I was thinking of something else entirely... well, never mind).
As far as I now understand in the case of PortProton there was a request for auth data to login Steam?
Are you running the Windows version of Steam? Then no wonder you received a auth request. Why would you run Steam for PortProton at all? You only need to run xash3d through PortProton.

**Comment by Nord1cWarr1or** [comment]
> How do you mean that you started the game with PortProton?
All I see in PortProton, regarding HL RTX, is how to start Steam, which I tried but Steam needs me to log in.

What's that got to do with steam? You only need Steam (native) to have the game. Install it. And all you have to do is run the game from the `*Steam/steamapps/common/Half-Life/rt_bin/xash3d.exe` through PortProton

**Comment by addeps3** [comment]
> 

Yes, Ive been having problems with that. In the UI of PortPr. there's only the options of "Installed", etc, and I cant add executables on it. =/ 



> @addeps3 I guess I just misunderstood you. Also the translation was wrong (but it doesn't matter, I was thinking of something else entirely... well, never mind). As far as I now understand in the case of PortProton there was a request for auth data to login Steam? Are you running the Windows version of Steam? Then no wonder you received a auth request. Why would you run Steam for PortProton at all? You only need to run xash3d through PortProton.

No idea, I studied the UI of PortPr. and the only options where Installed, Auto-install(?) etc. 
For the purpose of adding the xash3d.exe there was no functioning mechanism to do that. 
Maybe it was a problem of my build, I dont recall.

**Comment by Nord1cWarr1or** [comment]
@addeps3 Right-click on `xash3d.exe`, select "open via..." and select PortProton

**Comment by nonetrix** [comment]
I got it to at least compile natively on Linux somewhat but I think not fully. Should be 100% doable we have fairly good RT support now, and everything needed already supports Linux it seems. I will put more effort into getting it working when I feel like it, just remind me. I think someone else already got it working I saw on YouTube, but they didn't say how or anything
